eZ/Publish/Core/REST/Server/Tests/Output/ValueObjectVisitor/RestExecutedViewTest.php

```diff
@@ -140,6 +131,14 @@ public function getContentTypeServiceMock()
         return $this->createMock(ContentTypeService::class);
     }
 
+    /**
+     * @return \eZ\Publish\Core\Repository\LocationResolver\LocationResolver|\PHPUnit\Framework\MockObject\MockObject
+     */
+    public function getLocationResolverMock()
```

Comment:
If you only need LocationResolver interface, and it seems that this is the case, this can be typehinted to it.

Reply:
Resolved in https://github.com/ezsystems/ezpublish-kernel/pull/3055/commits/228a39a4a688a6aef399d58e8a5751d902ed3b5e